xenbus/xb.mli

```diff
@@ -137,6 +137,8 @@ val has_output : t -> bool
 
 val peek_output : t -> Packet.t
 
+val unsafe_pop_output : t -> Packet.t
```

Comment:
The binding in the mli doesn't look any different from the other bindings, when it should not be used in nornmal conditions. One way to make it stand out while reading the mli would be to put it at the end of the file, below a comment explaining the rest of exposed bindings is for testing only. A way to make it stand out while using autocompletion would be add a doc-comment with a warning that it must be used only for testing.